components/script/dom/htmlscriptelement.rs

```diff
@@ -508,7 +512,7 @@ impl HTMLScriptElement {
         let window = window_from_node(self);
         rooted!(in(window.get_cx()) let mut rval = UndefinedValue());
         window.upcast::<GlobalScope>().evaluate_script_on_global_with_result(
-            &script.text, script.url.as_str(), rval.handle_mut());
+            &script.text, script.url.as_str(), rval.handle_mut(), self.line_number as u32);
```

Comment:
We made the line numbers u64 in html5ever, so there's going to be a cast _somewhere_ until we change that.